src/secw_producer_accessor.cc

```diff
@@ -229,8 +229,6 @@ std::mutex g_lock;
 
 void callbackCreate(const std::string& portfolio, secw::DocumentPtr newDoc)
 {
-  std::cerr << "callback CREATED" << std::endl;
```

Comment:
This one could be replace by log_debug to keep tracing in the unit tests. Could you also use condition_variable for those instead of unlock/lock